content/blog/2019-10-23-loop-unrolling-with-bril.md

```diff
@@ -0,0 +1,154 @@
++++
+title = "Loop Unrolling with Bril"
+extra.author = "Shaojie Xiang & Yi-Hsiang Lai & Yuan Zhou"
+extra.bio = """
+  [Shaojie Xiang](https://github.com/Hecmay) is a 2nd year ECE PhD student researching on programming language and distributed system. 
+  [Yi-Hsiang (Sean) Lai](https://github.com/seanlatias) is a 4th year PhD student in Cornell's Computer System Lab. His area of interests includes electronic design automation (EDA), asynchronous system design and analysis, high-level synthesis (HLS), domain-specific languages (DSL), and machine learning. 
+  [Yuan Zhou](https://github.com/zhouyuan1119) is a 5th year PhD student in ECE department Computer System Lab. He is interested in design automation for heterogeneous compute platforms, with focus on high-level synthesis techniques for FPGAs.  
+"""
++++
+
+## Project Report 2: Loop Unrolling with Bril
```

Comment:
No need to include your title twice. (The template displays your `title` from above already.)